src/main/java/org/example/lesson4/util/ConnectionUtil.java

```diff
@@ -0,0 +1,53 @@
+package org.example.lesson4.util;
+
+import org.example.lesson4.exception.DataProcessingException;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.sql.ResultSet;
+
+public class ConnectionUtil {
+    private static final String DB_URL = "jdbc:sqlite:mydatabase.db";
+
+    public static Connection getConnection() {
+        try {
+            return DriverManager.getConnection(DB_URL);
+        } catch (SQLException e) {
+            throw new DataProcessingException("Unable get connection to DB", e);
+        }
+    }
+
+    public static void migration(){
+        Connection connect = getConnection();
+
+        String checkTableQuery = "SELECT name FROM sqlite_master WHERE type='table' AND name=?";
+
+        try (PreparedStatement prStatement = connect.prepareStatement(checkTableQuery)) {
+
+            String tableName = "users";
+            prStatement.setString(1, tableName);
+
+            String createTableQuery = "CREATE TABLE users (id INTEGER PRIMARY KEY, email VARCHAR(225),phoneNumber VARCHAR(15), password VARCHAR(20), repeatPassword VARCHAR(20))";
+
```

Comment:
Not sure it is a scalable solution that would be easy to support in the future. But for testing purpose it is ok.